src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -4636,14 +4612,52 @@ int wm_vuldet_get_software_info(agent_software *agent, sqlite3 *db, time_t min_f
             }
         }
         sqlite3_exec(db, vu_queries[END_T], NULL, NULL, NULL);
-        agent->info = 1;
-    } else if (*request == VU_SOFTWARE_REQUEST) {
-        mtdebug1(WM_VULNDETECTOR_LOGTAG, VU_NO_SOFTWARE, atoi(agent->agent_id));
-        if (agent->dist != FEED_WIN) {
-            // If no new inventory is available, skip the agent
-            // Only for Linux agents, on Windows the hotfixes scan may be enabled
-            retval = 2;
-            goto end;
+        scan_ctx->package_scan = TRUE;
+    }
+
+    // Operating System
+    if (VU_PARTIAL_SCAN != scan_ctx->scan_type  || 1 == wdb_agents_sys_osinfo_check_triaged(scan_ctx->agent_id, &sock))  {
```

Comment:
If we are checking the need for a OS scan, shouldn't be **0** the triaged value we are looking for ?

Reply:
It's true, this was a missing testing value. Nice catch!

Reply:
Thank you !